src/main/java/ch/uzh/ifi/hase/soprafs22/controller/UserController.java

```diff
@@ -187,4 +188,14 @@ public List<UserGetDTO> getMatches(
 
       return userGetDTOs;
   }
+
+  @PostMapping("/users/demo")
+  @ResponseStatus(HttpStatus.CREATED)
+  @ResponseBody
+  public void createDemoUsers(
+          @RequestHeader(value = "authorization", required = false) String token){
```

Comment:
What is the reason for the authorization header here?

Reply:
there's not really a good reason. i removed it